style.css

```
@@ -0,0 +1,212 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 100vh;
+  background: #131313;
+}
+
+.container .card {
+  position: relative;
+  overflow: hidden;
+
+  height: 450px;
+  width: 320px;
+  background: #232323;
+  border-radius: 20px;
+}
+
+.container .card:before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  height: 100%;
+  width: 100%;
+  background: #9bdc28;
+
+  clip-path: circle(150px at 20% 20%);
+
+  transition: 0.5s ease-in-out;
+}
+
+.container .card:hover:before {
+  clip-path: circle(300px at 20% -20%);
+}
+
+.container .card::after {
+  content: "Nike";
+  position: absolute;
+  top: 30%;
+  left: -20%;
+
+  font-style: italic;
+  font-size: 12em;
+  font-weight: 800;
+  color: rgba(255, 255, 255, 0.04);
+}
+
+.container .card .imgBx {
+  position: absolute;
+  top: 50%;
+
+  width: 100%;
+  height: 220px;
+
+  z-index: 100;
+  transform: translateY(-50%);
+  transition: .5s;
+}
+
+.container .card:hover .imgBx{
+    top: 0;
+    transform: translateY(0%);
+}
+
+.imgBx img{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+
+    width: 270px;
+
+    transform: translate(-50%,-50%) rotate(25deg);
+}
+
+.card .contentBx{
+    position: absolute;
+    bottom: 0;
+
+    width: 100%;
+    height: 100px;
+
+    text-align: center;
+
+    z-index: 10;
+    transition: 1s;
+}
+
+.card:hover .contentBx{
+    height: 210px;
+}
+
+.contentBx h2{
+    color: #fff;
+    font-weight: 600;
+    letter-spacing: 1px;
+}
+
+.contentBx .size,
+.contentBx .color{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 8px 20px;
+
+    transition: .5s;
+
+    opacity: 0;
+    visibility: hidden;
+}
+
+.card:hover .contentBx .size{
+    opacity: 1;
+    visibility: visible;
+    transition-delay: .5s;
+}
+
+.card:hover .contentBx .color{
+    opacity: 1;
+    visibility: visible;
+    transition-delay: .6s;
+}
+
+.size h3,
+.color h3{
+    color: #fff;
+    font-weight: 300;
+    font-size: 14px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+
+    margin-right: 10px;
```

```suggestion
    margin: 0 10px 0 0;
```
It is good to use shorthand.